packages/patternfly-4/react-core/src/components/Form/index.d.ts

```diff
@@ -0,0 +1,3 @@
+export { default as Form, FormProps } from './Form';
```

Comment:
Probably missed to add `export * from './Form';` into `packages/patternfly-4/react-core/src/components/index.d.ts` as well.

Reply:
added!